src/LendingPool.sol

```diff
@@ -865,6 +863,19 @@ contract LendingPool is LendingPoolGuardian, Creditor, DebtToken, InterestRateMo
         emit FixedLiquidationCostSet(fixedLiquidationCost_);
     }
 
+    /**
```

Comment:
detail, but we always use the same order:
- @notice
- @param
- @return
- @dev

Reply:
done